examples/contracts/README.md

```diff
@@ -0,0 +1,45 @@
+# Contracts
+
+Contracts Used for AA SDK
+
+Built using foundry + Alchemy
+
+- Foundry Getting Started: https://book.getfoundry.sh/getting-started/installation
+- Alchemy Getting Started: https://docs.alchemy.com/reference/api-overview
+
+## Index
+
+1. `./DAAppNFT` - A simple NFT based off the tutorial: https://book.getfoundry.sh/tutorials/solmate-nft
+
+## Development
```

Comment:
might be useful to include a bit about using `anvil` as a means of running a local node